tests/test_integration.py

```diff
@@ -76,14 +77,37 @@ def test_benchmark_models(
     state_space_functions,
     load_example_model,
 ):
-    params, options = load_example_model(f"{model}")
-    options["n_exog_states"] = 1
+    options = {}
+    params, _options = load_example_model(f"{model}")
+    _options["n_exog_states"] = 1
+
+    options["model_params"] = _options
+    options.update(
+        {
+            # "exogenous_processes": {
+            #     "exog": _return_one,
+            # },
+            "state_variables": {
+                "endogenous": {
+                    "period": jnp.arange(2),
```

Comment:
Our code has on multiple places a reliance that periods start at 0 and choices start at 0. In these tests, we assign n_periods and n_choices in "model_params". In solve we even rely on that. Lets not do that. Lets require, that the dictionary under the key "state_variables" (which I would name "state_space") has the variables n_periods and n_choices. And then optional "endogenous_states" and "exogenous_processes".

 This also explains why our code coverage goes down. You set the choices here to [0, 1], which is wrong in the deaton case.